.github/workflows/ci-workflows.yaml

```diff
@@ -0,0 +1,34 @@
+name: CI
+on:
+  push:
+    branches:
+    - master
+    - /^v[0-9]+\.[0-9]+\.[0-9]+$/ # version tags for Documenter
+    - /^release-.*$/ # Release branches, if any.
+  pull_request:
+    branches:
+    - master
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: [3.8]
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: "Set up Python ${{ matrix.python-version }}"
+        uses: actions/setup-python@v5
+        with:
+          version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements_dev.txt
```

Comment:
What is `requirements_dev.txt`?

Reply:
Good catch! Seems suspect uses only `requirements.txt` — I will fix this.

Some package maintains a separate `requirements_dev.txt`, which contains more dependencies that are only relevant to the development environment.